docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -590,30 +592,32 @@ After KYC is collected, you should be ready to receive the off-chain deposit. Wh
 
 </CodeExample>
 
-Amount should be collected in the interactive flow. However, it can also be specified by wallet as part of the request (see `amount_expected` of `GET /transaction/:id`). Please note, that you **must** specify amount fields, as wallet do not provide this information.
+The amount should be collected in the interactive flow. However, it can also be specified by the wallet as part of the request (see `amount_expected` of `GET /transaction/:id`).
+
+Please note that you **must** specify amount fields, as wallets do not provide this information.
 
-- `amount_in` is amount user requested. If `amount_expected` is not null, `amount_in` should be equal to `amount_expected`. In that case, amount should not be collected in the interactive flow.
-- `amount_out` is amount user will receive
-- `amount_fee` is total fees collected by the application.
-- `asset` part of amount fields is an [SEP-38][sep-38] formatted asset of the amount. In this example, it's set to USD, assuming user made a bank transfer to the system using USD.
+- `amount_in` is the amount the user requested. If `amount_expected` is not null, `amount_in` should be equal to `amount_expected`. In that case, the amount should not be collected in the interactive flow.
+- `amount_out` is the amount the user will receive.
+- `amount_fee` is the total fees collected by the application.
+- `asset` is part of the amount fields is a [SEP-38][sep-38] formatted asset of the amount. In this example, it's set to USD, assuming the user made a bank transfer to the system using USD.
 
 :::tip
 
-Asset field is optional. When not set, it defaults to the asset user requested.
+The Asset field is optional. When not set, it defaults to the asset the user requested.
 
 :::
 
 ##### Waiting For User Funds
 
 :::tip
 
-This step is optional. Most business don't use it: you can skip it and go to the [next one](#sending-stellar-transaction)
+This step is optional. Most businesses don't use it: you can skip it and go to the [next step](#sending-stellar-transaction)
 
 You should only use this status if the off-chain payment method being used notifies you of a pending inbound payment.
 
 :::
 
-After update in the previous chapter has been made, user will be notified that your service is ready to receive funds. In real world, transfer confirmation may take time. In such cases, transaction should be set to a new status, after funds have been received, but not confirmed yet.
+After the update in the previous chapter has been made, the user will be notified that your service is ready to receive funds. In the real world, transfer confirmation may take time. In such cases, transaction should be set to a new status, after funds have been received, but not confirmed yet.
```

Comment:
Wording here is also a little confusing. Does this mean that the funds have been received but not confirmed? If so, maybe we can say: "In the real world, transfer confirmation may take time. If the funds have been received but not confirmed, the transaction should be set to a new status."

Reply:
Yeah, this part is worded incorrectly. It means:
Transfer is in process. Confirmation of the transfer has been received, but funds itself may not been received yet